cluster/gce/gci/configure-helper.sh

```
@@ -25,6 +25,9 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+### Hardcoded constants
+METADATA_SERVER_IP="169.254.169.254"
```

It's flexible to allow overriding but default to the hardcoded IP:

```
METADATA_SERVER_IP="${METADATA_SERVER_IP:-169.254.169.254}"
```